minerl/Malmo/Schemas/MissionHandlers.xsd

```diff
@@ -1636,13 +1636,13 @@
             <xs:documentation>
                 A command for simple smelting:
 
-                "{{{ smeltNearby carpet PINK}}}"
+                "{{{ nearbySmelt carpet PINK}}}"
```

Comment:
Interesting, so this was the same bug as the one we had with craftNearby/nearbyCraft, but it seems we are choosing opposite solutions.

Note that I do not really have a strong opinion on this, and I think Brandon recommended that I go with "craftNearby", but I can also see an argument for "nearbyCraft" because that is the name of the object in Java.

Either way, I think our solution should be consistent.

Reply:
Good point, I did not compare it to craftNearby / nearbyCraft. Will fix to the craft standard.

Reply:
I'm with changing craftNearby as well, the less ambiguity here the better

Reply:
uh I am confused now, which version would you guys prefer me to follow? Right now it is changed so that java commands are called smeltNearby and craftNearby, is that fine?